air-quality-backend/scripts/run_forecast_etl.py

```diff
@@ -33,6 +33,12 @@ def main():
     logging.info("Persisting forecast data")
     insert_data(transformed_forecast_data)
 
+    logging.info("Creating forecast data textures")
+    textures = create_data_textures(extracted_forecast_data)
```

Comment:
Would be good to have another chat about this at some time, because I am not sure I understand this correctly. Let's leave this for the API ticket as suggested?

Reply:
Yep no problem